Mathlib/RingTheory/Jacobson/Semiprimary.lean

```diff
@@ -45,6 +45,10 @@ theorem IsSemisimpleModule.jacobson_le_annihilator [IsSemisimpleModule R M] :
     have := Module.le_comap_jacobson (LinearMap.toSpanSingleton R M m) hr
     rwa [jacobson_eq_bot] at this
 
+instance (priority := 900) (R) [CommRing R] [IsSemisimpleRing R] : IsReduced R where
```

Comment:
```suggestion
instance (priority := low) (R) [CommRing R] [IsSemisimpleRing R] : IsReduced R where
```
Seems the standard way, unless you have a specific reason for `900`.

Reply:
I'm following Andrew's [advice](https://github.com/leanprover-community/mathlib4/pull/18686#discussion_r1835316469), though I'm not exactly sure whether it applies here.

Reply:
I think `low` might be the canonical way.